models/marketing/intermediate/int_hubspot__email_metrics__by_contact_list.sql

```diff
@@ -14,7 +14,8 @@ with email_sends as (
 
     select
         email_sends.*,
-        contact_list_member.contact_list_id
+        contact_list_member.contact_list_id,
+        coalesce(contact_list_member.is_contact_list_member_deleted, false) as is_contact_list_member_deleted
```

Comment:
what's the purpose of bringing in the is_contact_list_member_deleted field here, since it's in aggregational model? do we need to bring it into the contact_list end model or sum up the email_metrics again there to avoid fanout? i am getting some errors when running `dbt test`

```
Completed with 11 errors and 0 warnings:
21:10:22  
21:10:22  Failure in test unique_hubspot__contact_lists_contact_list_id (models/marketing/marketing.yml)
21:10:22    Got 7 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/marketing.yml/unique_hubspot__contact_lists_contact_list_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_bounce_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 14 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_bounce_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_clicks_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 7 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_clicks_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_deferred_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 4 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_deferred_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_delivered_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 336 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_delivered_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_dropped_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 42 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_dropped_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_opens_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 1218 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_opens_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_status_change_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 2 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_status_change_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_event_sent_event_id (models/marketing/email_events/email_events.yml)
21:10:22    Got 724 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/email_events/email_events.yml/unique_hubspot__email_event_sent_event_id.sql
21:10:22  
21:10:22  Failure in test unique_hubspot__email_sends_event_id (models/marketing/marketing.yml)
21:10:22    Got 724 results, configured to fail if != 0
21:10:22  
21:10:22    compiled Code at target/compiled/hubspot/models/marketing/marketing.yml/unique_hubspot__email_sends_event_id.sql
21:10:22  
21:10:22  Failure in test unique_int_hubspot__email_metrics__by_contact_list_contact_list_id (models/marketing/intermediate/intermediate.yml)
21:10:22    Got 7 results, configured to fail if != 0
```

Reply:
hm I think you're right here! I think when I started I just followed along with [the plan](https://github.com/fivetran/dbt_hubspot_source/issues/87#issuecomment-1315674869) but I agree, I don't think it's actually necessary to include it here.